locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -80,11 +80,11 @@ touch Dockerfile
 Open the `Dockerfile` in your favorite text editor
 
 The first thing we need to do is define from what image we want to build from.
-Here we will use the latest LTS (long term support) version `8` of `node`
+Here we will use the latest LTS (long term support) version `10` of `node`
 available from the [Docker Hub](https://hub.docker.com/):
 
 ```docker
-FROM node:8
+FROM node:10
```

Comment:
this _could_ be `node:lts` instead of hard coding a version number, not sure if it should be recommended or not, though

Reply:
No it should not. It's problematic in the same way `latest` is.

Reply:
We're currently having this discussion in PR #2953. Can you elaborate on this? I clearly see there might be issues using the `latest` tag, but I don't understand in what way this might be problematic in the same way `latest` would be.